### PR TITLE
Kade Engine Android Pog?

### PR DESCRIPTION
**Pull request by ReylordFX**
Poggers lul

### CODE REVIEW

**Comment by The-SGPT** [comment]
uhhhh i dont think ninjas going to pull all these commits

**Comment by BoyfriendFNF** [comment]
I want this to happen.

**Comment by KadeDev** [comment]
dude ninja aint gonna pull this lol!

File: source/FreeplayState.hx [hunk 1]
```diff
@@ -47,6 +52,11 @@ class FreeplayState extends MusicBeatState
 			}
 		 */
 
+		 #if desktop
+		 // Updating Discord Rich Presence
+		 DiscordClient.changePresence("In the Menus", null);
```
Comment (actuallyasmartname) [inline source/FreeplayState.hx:57]: Maybe set this to "In the Freeplay menu" for Freeplay and "In the story mod menu" for story mode?

**Comment by Riteo** [comment]
Bruh, the title mentions an android port, yet I see like 3000 other changes. You should split it up. Also, you should rebase. 103 commits are not poggers.

**Review by windows10win7posreadywoeskio — APPROVED** [review]

**Comment by PrincessMtH** [comment]
yo we really just out here opening pull requests to merge other people's code?

**Comment by hdboye** [comment]
step 1 open da pr
step 2 link popular community member's code
step 3 profit

**Comment by hdboye** [comment]
btw please close. this is literally just kade engine.

**Comment by actuallyasmartname** [comment]
Please close this. This has a lot of other people's code and has way too much commits to even be just a Kade Engine for android

**Comment by Squiddu** [comment]
Doubt they'll do anything now, this is like the second time this has happened

**Comment by CyndaquilDAC** [comment]
Seriously? Trying to pull Kade Engine?

**Comment by VoltrexMaster** [comment]
:trollface: troll PR much?

**Comment by hdboye** [comment]
man made an account, made a pr, and left :trollface:

**Comment by aflacc** [comment]
bruhhhh

**Comment by Squiddu** [comment]
Someone please end this madness

**Comment by aflacc** [comment]
:trollface: please end my suffering

**Comment by actuallyasmartname** [comment]
:trollface: bruh

**Comment by aflacc** [comment]
please stop your scaring me

**Comment by aflacc** [comment]
HELP

**Comment by aflacc** [comment]
help

**Comment by aflacc** [comment]
i regret talking here

**Comment by nickpythonluajava** [comment]
WHOOO YEAH BABY THATS WHAT I'VE BEEN WAITING FOR THAT'S WHAT IT'S ALL ABOUT

**Comment by Steviegt6** [comment]
Absolutely cursed pull request.

**Comment by VMGuy23** [comment]
most files conflict :) and watching this repo hurts because of this
@ninjamuffin99 please close this

**Pull request closed without merging**